circuits/circom/core/non-qv/processMessages.circom

```diff
@@ -169,50 +161,23 @@ include "../../trees/incrementalQuinaryTree.circom";
 
     // Hash each Message to check their existence in the Message tree.
     var computedMessageHashers[batchSize];
+    var computedHashes[batchSize];
+    var chainHash[batchSize + 1];
+    chainHash[0] = inputBatchHash;
     for (var i = 0; i < batchSize; i++) {
```

Comment:
let's add comments to explain what is happening in this loop please

Reply:
resolved